Update icons, `ru_ru.json`

mpustovoi opened the pull request:
- `ru_ru.json`:
  - Added missing lines.
- Icons:
  - Added icons for Golem of Wrath, Living Armor and Sculk Guardian.
  - The rest of the files were just automatically formatted by VS Code.

mpustovoi committed:
Update `en_us.json`

Added missing lines with the names for the dev stuff so it's displayed normally in JEI/EMI.

mpustovoi committed:
Update `ru_ru.json`

Parity with `en_us.json`.

mpustovoi committed:
Added icons for dev mobs

Added icons for Golem of Wrath, Living Armor and Sculk Guardian.
The rest of the files were just automatically formatted by VS Code.

mpustovoi committed:
Merge branch 'TeamPeril:1.20' into patch-1

mpustovoi committed:
Update `en_us.json`

Adapted to v0.9.26

mpustovoi committed:
Update `ru_ru.json`

Adapted to v0.9.26

Mikeatron-User commented:
The names and textures of things are not yet finalized, so I wont be merging this until later on.

mpustovoi committed:
Updated icons

Updated Soul Reaper icon, added beta icons for Sculk Brood Hatcher & Sculk Broodling.

mpustovoi committed:
Update `ru_ru.json`

v0.9.31

mpustovoi committed:
Updated icons

Updated Sculk Brood Hatcher icon.

mpustovoi committed:
Updated icons

Added Sculk Sheep icon.

mpustovoi committed:
Update `ru_ru.json` & icon

- Updated `ru_ru.json`.
- Updated Sculk Sheep icon.

mpustovoi committed:
Update `ru_ru.json`

v0.9.40

mpustovoi committed:
v0.10.0

- Tweaked Soulite translation.
- Added icons for Sculk Leech & Stinger.
